### DISCUSSION TITLE
[FEATURE]: Copy/clone a listing

### DISCUSSION THREAD

**Discussion by nctbrtc** [etsy/open-api#719] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

It would be very nice if we can have a copy feature like we have on the web UI.
That's useful for the shop owner who sells items like tshirts 

### Describe the solution you'd like

We will provide the **listing id** we can to copy along with;
-new title
-new desc
-new tags
-new image ids

and it will copy the listing with our new inputs and will response with the new listing id we can use

**Comment by etsyachristensen** [comment]
Thanks for the suggestion @nctbrtc . We have it on our list of future development possibilities.

**Comment by etsyachristensen** [comment]
We're converting this listing to an Idea Discussion for continued tracking/

**Comment by simonj** [comment]
do we have this now? I can't seem to find anything on the API doc 

cc @nateswart

**Reply by Gareth064** [reply]
We do not. It never materialized.

But I do wonder why we would need Etsy to build this. We have the API to get all the details of a current listing, so why not do that then using the data from it, create your new listing.

**Reply by simonj** [reply]
@Gareth064 You are right. I just found this https://developers.etsy.com/documentation/reference/#operation/getListing

**Reply by simonj** [reply]
But again, we can't just do something like this

```php
$etsyService->createDraftListing($etsyService->getListing($listing->external_listing_id));
```

And again we would have to manipulate the response we get from getListing
